README.md

```diff
@@ -39,21 +39,72 @@ pod:
 
 # Deployment
 
-*Warning:* Deploying the cilium-etcd-operator will automatically overwrite the
-Kubernetes secret `cilium-etcd-secrets`. If you have configured Cilium to use
-an external etcd, it is likely using the same secret name so deploying the
-cilium-etcd-operator will overwrite that secret.
+Deploying the cilium-etcd-operator will automatically only create the Kubernetes
+secret `cilium-etcd-secrets` if it does not exist. If you have configured Cilium
+to use an external etcd, it is likely using the same secret name so deploying the
+cilium-etcd-operator will not overwrite that secret.
 
-If you already have a mechanism to generate certs and to avoid fresh certificate generation/overwrites to existing ones, set the following environment variable:
+If you want to overwrite the certificates every time you restart
+`cilium-etcd-operator` set the following environment variable:
 
 ```
-CILIUM_ETCD_OPERATOR_GENERATE_CERTS=false
+        - name: CILIUM_ETCD_OPERATOR_GENERATE_CERTS
+          value: "false"
```

Comment:
Does this need to be changed to true in order to enable the overwrite?

Reply:
yep, good catch!